### ISSUE TITLE
SassError: Invalid CSS after " ": expected 1 selector or at-rule, was "{" 

### ISSUE THREAD

**Issue by panhaoyu** [vuejs/vue-loader#1707]
### Version
16.0.0-beta.4

### Reproduction link
[https://github.com](https://github.com)

### Steps to reproduce
`vue create site`
choose vue3 and sass
change App.vue style like this:

```
<style lang="sass">
    #app
        font-family: Avenir, Helvetica, Arial, sans-serif
        -webkit-font-smoothing: antialiased
        -moz-osx-font-smoothing: grayscale
        text-align: center
        color: #007dff
        margin-top: 60px
</style>
```

make sure that `#App` has 4 space indent.

npm run serve, error.

### What is expected?
serve

### What is actually happening?
abrupt

---
this is an available work around: add a line in `node_modules/sass-loader/dist/index.js`:

```
function loader(content) {
    const options = (0, _loaderUtils.getOptions)(this);
```

to 

```
function loader(content) {
    content = content.replace('\r\n', '\n')
    const options = (0, _loaderUtils.getOptions)(this);
```

**Comment by panhaoyu** [comment]
Sorry no repo, but I'm not major in computer science, and I do not have enough time to do more.

**Comment by panhaoyu** [comment]
And I don't think this is a sass-loader error, for it works well, if vue provide correct code.

**Comment by panhaoyu** [comment]
After I changed to dart-sass, everything works well.

**Comment by sodatea** [comment]
> Sorry no repo, but I'm not major in computer science, and I do not have enough time to do more.

Sorry, but if you don't have the time to create a simple reproduction repo, we as open-source maintainers don't have the time to create a new project just to reproduce a "possible" bug either.

**Comment by panhaoyu** [comment]
yes I know, I thought for a while if I should open this issue, but I thought open an issue is better than nothing, at least for others to search this issue and locate the problem

**Comment by panhaoyu** [comment]
BTW I'm really sorry and I do not attend to offend open source maintainers, thanks for your work
